src/topp/PercolatorAdapter.cpp

```diff
@@ -229,6 +229,8 @@ class PercolatorAdapter :
     registerFlag_("peptide-level-fdrs", "Calculate peptide-level FDRs instead of PSM-level FDRs.");
     registerFlag_("protein-level-fdrs", "Use the picked protein-level FDR to infer protein probabilities. Use the -fasta option and -decoy-pattern to set the Fasta file and decoy pattern.");
     registerStringOption_("osw_level", "<osw_level>", "ms2", "OSW: Either \"ms1\", \"ms2\" or \"transition\"; the data level selected for scoring.", !is_required);
+    registerStringOption_("score_type", "<type>", "q-value", "Type of the peptide main score", false);
+    setValidStrings_("score_type", ListUtils::create<String>("q-value,Posterior Error Probability,SVM Score"));
```

Comment:
Maybe the input options would be better without spaces? E.g. "qvalue", "pep", "svm"? This makes it easier to pass the parameters via CLI.